tests/unit/SolrUpdateTest.php

```diff
@@ -16,13 +20,24 @@ class SolrUpdateTest extends SapphireTest
      * @var SolrUpdate
      */
     protected $solrUpdate;
+    public static function setUpBeforeClass()
+    {
+        parent::setUpBeforeClass();
+        // This is a hack on my local dev to make fixtures populate temporary database
+        $conn = DB::get_conn();
+        $dbName = self::tempDB()->build();
+        $conn->selectDatabase($dbName);
+        $dbAdmin = new DatabaseAdmin();
+        $dbAdmin->setRequest(HTTPRequestBuilder::createFromEnvironment());
+        $dbAdmin->doBuild(true, true, true);
+    }
 
     /**
      * @expectedException \LogicException
      */
     public function testUpdateItemsFail()
     {
-        $this->solrUpdate->updateItems(null, SolrUpdate::CREATE_TYPE);
+        $this->solrUpdate->updateItems(null, SolrUpdate::CREATE_TYPE, 'unknown');
```

Comment:
This should always fail at the check in the updateItems (as noted above)

Reply:
Cool, will rebase the code.